utils/ort/src/test/kotlin/OkHttpClientHelperTest.kt

```diff
@@ -58,6 +59,24 @@ class OkHttpClientHelperTest : WordSpec({
 
             clientA shouldBeSameInstanceAs clientB
         }
+
+        "return different clients when specifying different configurations" {
+            val timeoutA: BuilderConfiguration = { readTimeout(Duration.ofSeconds(100)) }
+            val timeoutB: BuilderConfiguration = { readTimeout(Duration.ofSeconds(500)) }
+            val clientA = OkHttpClientHelper.buildClient(timeoutA)
+            val clientB = OkHttpClientHelper.buildClient(timeoutB)
+
+            clientA shouldNotBeSameInstanceAs clientB
+        }
+
+        "return different clients when specifying different configurations instances with the same values" {
```

Comment:
configurations instances -> configuration instances?

Reply:
I've reworded this yet more to emphasize that differently clients are being returned despite the configuration being the same (value-wise).